### DISCUSSION TITLE
Ability to set nodejs path in nvim

### DISCUSSION THREAD

**Discussion by adikari** [community/community#13310] [Copilot Conversations] [answered]
I am running nvim in mac with m1 chip. Seems like the minimum node version required is 16. I work on multiple projects which require node js 14. I am setting those node versions using `fnm`. 

For my project requirements, I need to be on node 14. Would be nice to add the ability to configure the nodejs path used by the copilot. Something like `let g:copilot_nodejs_executable = 'path to nodejs binary'`.

I checked the plugin code and seems like it's hardcoded to `node` which uses whatever node is set to the terminal session. 

Let me know if there is any other to work around this problem.

**Comment by tpope** [comment, marked as answer]
There is an undocumented option:

```vim
let g:copilot_node_command = 'path to nodejs binary'
```

**Reply by Jeanno** [reply]
works perfectly
